.github/workflows/pr-validation.yml

```diff
@@ -11,22 +11,25 @@ jobs:
         name: Install Node 16
         with:
           node-version: '16'
-          cache: 'yarn'
+          cache: 'npm'
+      - name: Install netlify-cli
+        run: npm install --save-dev @nx/nx-linux-x64-gnu
```

Comment:
This installs `nx` not netlify

Reply:
`nx` also needs to be installed in the CI, I just labelled it wrong. I'll fix it